teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(42, 'Stephen')
```

Comment:
- [ ] The teacher class extends from the person class so it should also require the arguments used in creating a person.

  ```
  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  ```